### PR TITLE
Fix most vulnerabilities and some standardjs fixes

### PR DESCRIPTION
**Pull request by juliedavila**
Currently running `npm audit` on this project shows
```found 20 vulnerabilities (11 low, 6 moderate, 2 high, 1 critical) in 856 scanned packages
  18 vulnerabilities require semver-major dependency updates.
  2 vulnerabilities require manual review. See the full report for details.
```
This PR fixes all of them except the 2 manual review findings which are moderate. I'll be making a PR to `explicit` soon which is where the `hoek` vulnerability currently exists.

This also fixes some standardjs style issues.

This does not break commandico tests. After applying the update the tests show
```> commandico@2.0.2 test /home/jdavila/Repos/commandico
> npm run lint; mocha


> commandico@2.0.2 lint /home/jdavila/Repos/commandico
> standard



  standard execution
    ✓ simplest execution
    ✓ fallback execution
    ✓ fallback on missing array
    ✓ fallback on null
    ✓ fallback on something
    ✓ non default entries
    ✓ non default entries
    ✓ modifier execution
    ✓ missing modifiers
    ✓ missing default
    ✓ modifier filter=true execution
    ✓ modifier filter=false execution
    ✓ multiple modifiers
    ✓ menu filter
    ✓ menu filter order test
    ✓ command order test
    ✓ multi command order test
    ✓ loadCommands test
    ✓ ordinary modifier executuoin
    ✓ loadModifiers test
    ✓ ordered multi command order test
    ✓ add various orders
    ✓ add various orders


  23 passing (20ms)
```

### CODE REVIEW

**Comment by juliedavila** [comment]
@martinheidegger based on the lack of updates to this project for roughly two years, I understand if you no longer actively maintain this (maybe I'm wrong?). If you'd like, I'm happy to take the reins.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "commandico",
-  "version": "2.0.2",
+  "version": "2.0.3",
```
Comment (martinheidegger) [inline package.json]: Please don't change the version in a PR. The version should be changed on/before publish. Thank you for the consideration on my effort though! ❤️

File: test/commandico.js [hunk 2]
```diff
@@ -255,7 +256,7 @@ describe('standard execution', function () {
 
   it('loadCommands test', function (done) {
     var command = commandico(null, 'menu')
-      .loadCommands(__dirname + '/../test_cmds')
+      .loadCommands(path.join(__dirname, '/../test_cmds'))
```
Comment (martinheidegger) [inline test/commandico.js]: I recently learned that `path.join` internally uses `path.normalize` maybe better to use that?

**Review by martinheidegger — APPROVED** [review]
Thank you for this PR as well. Its awesome that you take the time on this. I noticed one little thing but other than that its fine. Please let me know if I should merge it.

Regarding maintainer-ship: I am not opposed to hand over maintainership to someone else. If you don't mind though: Lets first get a few PR's under our belly.

File: test/commandico.js (hunk 2 above)
Comment (juliedavila) [reply test/commandico.js]: I don't think one is better or worse, it's a preference thing. If you prefer normalize, sure, I'll update it.

**Comment by juliedavila** [comment]
alright, @martinheidegger your requested changes have been made, thank you!

**Comment by martinheidegger** [comment]
Published as 2.0.3

**Pull request merged**